### DISCUSSION TITLE
Incremental builds for Angular applications using custom-webpack builders

### DISCUSSION THREAD

**Discussion by Splaktar** [nrwl/nx#4896] [General]
Hello,

I've got a large project using `"@angular-builders/custom-webpack": "^11.0.0"` via
```json
        "build": {
          "builder": "@angular-builders/custom-webpack:browser",

        },
        "serve": {
          "builder": "@angular-builders/custom-webpack:dev-server",

        },
```

I was reading [Setup incremental builds for Angular applications](https://nx.dev/latest/angular/ci/setup-incremental-builds-angular) and saw that it requires using custom Nx builders:
```json
        "build": {
          "builder": "@angular-builders/@nrwl/angular:webpack-browser",

        },
        "serve": {
          "builder": "@nrwl/web:file-server",

        },
```

Is there any way to get incremental builds while also using the `custom-webpack` builder? 

Any plans or ideas for supporting this in the future?

If not, is there a way to use, or plans to support using, a custom Webpack config with the `@nrwl` builders?

**Comment by Jordan-Hall** [comment]
My thinking is we could pass in an option builder in the schema a bit likeI did for the Storybook builder to decide if it's react, angular, vue etc. Default the value to the angular builder. That way its backwards compatibility and we have more flexibility?

**Comment by Coly010** [comment]
There is also currently a PR to support custom webpack config :) 
https://github.com/nrwl/nx/pull/5609

**Reply by Serg-Mois** [reply]
@Coly010 do you have any examples of using it? Is it support typescript config? Why you didn't update documentation?
